src/MainWindow.cpp

```diff
@@ -359,6 +359,16 @@ MainWindow::event(QEvent *event)
     return QQuickView::event(event);
 }
 
+// HACK: https://bugreports.qt.io/browse/QTBUG-83972, qtwayland cannot auto hide menu
+void
+MainWindow::mousePressEvent(QMouseEvent *event)
+{
+    if (event->button() == Qt::LeftButton) {
```

Comment:
I'm assuming this does not break opening menus?

Reply:
I suspect it's wrong. It will call function each time when user clicked LB.

Reply:
Oh, auto-clicker...